<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>30</y>
      <width>1131</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(86, 197, 150);
</string>
    </property>
    <property name="title">
     <string>Gestion des employés</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(50, 157, 156);</string>
     </property>
     <property name="text">
      <string>Gestion d'employés</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 244, 210);</string>
     </property>
     <property name="text">
      <string>Gestion de Produits</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 244, 210);</string>
     </property>
     <property name="text">
      <string>Gestion de collectes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 244, 210);</string>
     </property>
     <property name="text">
      <string>Rapport et Analyse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 244, 210);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 244, 210);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 244, 210);</string>
     </property>
     <property name="text">
      <string>Accueil</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>901</width>
       <height>621</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget
{
background-color: rgb(123, 228, 149);

}

QTabWidget:tab-bar
{
alignment:center;
}

QTabBar:tab
{
width:100px;
height:50px;
}

QTabBar:tab:selected
{
    
	background-color: rgb(50, 155, 156);
    color:rgb(0,255,0);
}

QTabBar:tab:hover
{
	background-color: rgb(109, 202, 131);
}

</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ajouter </string>
      </attribute>
      <widget class="QPushButton" name="ajout">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>520</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>9</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(207, 244, 210);
border-radius: 10px;
}


QPushButton:hover {
background-color: rgb(50, 157, 156);

}</string>
       </property>
       <property name="text">
        <string>AJOUTER</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>461</width>
         <height>451</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid  rgb(139, 255, 51);
background-color: rgb(86, 197, 150);
border-radius: 10px;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nom et prénom : </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pnom">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>20</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="padresse">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>90</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Num téléphone :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Adresse Email :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pnumtel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>170</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CIN :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pcin">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>130</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date de recrutement :</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="pdate">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>210</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="pdepartement">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>250</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Département :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Salaire :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="psalaire">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>330</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="pposte">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>290</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mot de passe :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="mdp">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>50</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>380</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Heures de travail fixes :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>410</y>
          <width>291</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Heures de travail supplémentaires :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="phfixe">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>370</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="phsupp">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>410</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Poste :</string>
        </property>
       </widget>
      </widget>
      <widget class="QComboBox" name="selectii">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>60</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid  rgb(139, 255, 51);
background-color: rgb(86, 197, 150);
border-radius: 10px;
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Forte</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Gestion des employés</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>520</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(207, 244, 210);
border-radius: 10px;
}


QPushButton:hover {
background-color: rgb(50, 157, 156);

}</string>
       </property>
       <property name="text">
        <string>MODIFIER</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Afficher</string>
      </attribute>
      <widget class="QTableView" name="affichage">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>761</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid  rgb(139, 255, 51);
background-color: rgb(86, 197, 150);
border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid  rgb(139, 255, 51);
background-color: rgb(86, 197, 150);
border-radius: 10px;</string>
       </property>
       <item>
        <property name="text">
         <string>Nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Departement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poste</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="rechText">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid  rgb(139, 255, 51);
background-color: rgb(86, 197, 150);
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="chercher">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>9</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(207, 244, 210);
border-radius: 10px;
}


QPushButton:hover {
background-color: rgb(50, 157, 156);

}</string>
       </property>
       <property name="text">
        <string>CHERCHER</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PDF">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>490</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>9</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(207, 244, 210);
border-radius: 10px;
}


QPushButton:hover {
background-color: rgb(50, 157, 156);

}</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>530</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>9</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(207, 244, 210);
border-radius: 10px;
}


QPushButton:hover {
background-color: rgb(50, 157, 156);

}</string>
       </property>
       <property name="text">
        <string>Excel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="afficher">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>470</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>9</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(207, 244, 210);
border-radius: 10px;
}


QPushButton:hover {
background-color: rgb(50, 157, 156);

}</string>
       </property>
       <property name="text">
        <string>AFFICHER</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="supp">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>510</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid  rgb(139, 255, 51);
background-color: rgb(86, 197, 150);
border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>510</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>9</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(207, 244, 210);
border-radius: 10px;
}


QPushButton:hover {
background-color: rgb(50, 157, 156);

}</string>
       </property>
       <property name="text">
        <string>SUPPRIMER</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>510</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CIN :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="trier">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>510</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>9</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(207, 244, 210);
border-radius: 10px;
}


QPushButton:hover {
background-color: rgb(50, 157, 156);

}</string>
       </property>
       <property name="text">
        <string>TRIER</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>510</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid  rgb(139, 255, 51);
background-color: rgb(86, 197, 150);
border-radius: 10px;
</string>
       </property>
       <item>
        <property name="text">
         <string>Nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Departement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poste</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Salaire</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Statistique</string>
      </attribute>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>871</width>
         <height>521</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_8">
        <attribute name="title">
         <string>Departement</string>
        </attribute>
        <widget class="QFrame" name="horizontalFrame">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>821</width>
           <height>431</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_9">
        <attribute name="title">
         <string>Poste</string>
        </attribute>
        <widget class="QFrame" name="horizontalFrame_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>831</width>
           <height>431</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="PICOOO">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../LOGO/gemstonerecycle.png</pixmap>
     </property>
    </widget>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>label_pic</zorder>
    <zorder>tabWidget</zorder>
    <zorder>pushButton_7</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>PICOOO</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
